woocommerce/woocommerce-paypal-payments
# Combine the webhooks status page into a new connection tab

**Issue**: #827

---

### Description

It is proposed that the Webhooks Status tab should be removed. The features provided in this tab should be moved to the new Connection tab.
The PR will move the "WebHook Status" tab settings into the new "Connection" tab.

### Steps to Test

1. Check the "Connection" tab settings and make sure the "Webhook Status" section exists.
2. Check if the "Webhook Status" tab is removed.

---

Closes #827 .
